src/main/java/org/team1540/robot2018/subsystems/Elevator.java

```diff
@@ -0,0 +1,76 @@
+package org.team1540.robot2018.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import org.team1540.base.ChickenSubsystem;
+import org.team1540.base.util.SimpleCommand;
+import org.team1540.base.wrappers.ChickenTalon;
+import org.team1540.robot2018.RobotMap;
+import org.team1540.robot2018.Tuning;
+import org.team1540.robot2018.commands.elevator.HoldElevatorPosition;
+
+public class Elevator extends ChickenSubsystem {
+
+  private ChickenTalon talon1 = new ChickenTalon(RobotMap.ELEVATOR_1);
+  private ChickenTalon talon2 = new ChickenTalon(RobotMap.ELEVATOR_2);
+
+  public Elevator() {
+    this.add(talon1, talon2);
+    this.setPriority(10);
+
+    talon1.configSelectedFeedbackSensor(FeedbackDevice.QuadEncoder);
+    Command command = new SimpleCommand("Zero Elevator", () -> talon1.setSelectedSensorPosition(0));
+    command.setRunWhenDisabled(true);
+    SmartDashboard.putData(command);
```

Comment:
uh why? Also adding stuff into SmartDashboard in a decentralized manner

Reply:
Allows zeroing of the elevator if it's out of alignment, whether due to just being turned on or something breaking.

Reply:
Centralize it then?

Reply:
Can do, I suppose.